pkg/utils/test_util.go

```
@@ -37,6 +47,29 @@ func GetEventString(object runtime.Object, eventtype, reason, messageFmt string,
 			object.GetObjectKind().GroupVersionKind().Kind, object.GetObjectKind().GroupVersionKind().GroupVersion())
 }
 
+// GetObjectFromRawExtension returns an object decoded from the raw byte array
+func GetObjectFromRawExtension(rawByte []byte, obj runtime.Object) {
+	json, err := yaml.ToJSON(rawByte)
+	if err != nil {
```

we should not ignore any error.